### PR TITLE
feat(login): begin adding account recovery page

### LINKED ISSUE #1181: task(global): Show and restore with recovery phrase
1. During the account creation, the UI should show the mnemonic phrase before proceeding to call `MultiPass::create_identity` to create the account (possibly even confirming the phrase in some way, see below), but can also show it after calling it from `IdentityProfile::passphrase`.
2. When restoring an keypair with the mnemonic phrase, it should be passed into `MultiPass::import_identity` or `MultiPass::create_identity`

Note:
1. ~~At this time, it will only restore the keypair and not the identity itself. Restoring identity itself will come later~~ You can restore the keypair with the phrase via `MultiPass::create_identity` or import the account via `MultiPass::import_identity`
2. ~~To generate a mnemonic phrase, call `warp::crypto::keypair::generate_keypair` and use the first item in the unit for the phrase (although this will change soon to only show the phrase from this function)~~ You can use `IdentityProfile` after creating the account to provide the mnemonic phrase if one was not supplied.
3. (Optional) When creating a new account, it might be wise to have then confirm their phrase after they confirm it was saved in some manner (note that at this time, recovery of the phrase is not possible, though a flag could be enabled to allow saving it in tesseract for revealing the phrase later for new accounts, it is, in my own opinion, best to leave it one way to reduce the possibility of the phrase being leaked somehow)

### PR DESCRIPTION
**Pull request by sdwoodbury**
### What this PR does 📖

- adds a MultiPassCmd to recover an account. 
- consolidates the login related pages under `layouts` into a single folder. 

### Which issue(s) this PR fixes 🔨

- Resolve #1181

### Additional comments 🎤

example way to test: ` RUST_LOG=warp_ipfs=debug,uplink=debug,common=debug target/debug/uplink --path=/tmp/warp2 --discovery shuttle`

for seed word generation, the [bip39](https://docs.rs/bip39/latest/bip39/) crate was used. This one seems to be the good one.

### CODE REVIEW

**Comment by sdwoodbury** [comment]
closing so ppl don't merge dev with it before it's open. will reopen when ready

File: Cargo.toml [hunk 1]
```diff
@@ -45,9 +45,9 @@ arboard = "3.2"
 humansize = "2.1.3"
 uuid = { version = "1", features = ["serde", "v4"] }
 libloading = "0.7.4"
-warp = { git = "https://github.com/Satellite-im/Warp", rev = "786328d0c7ef6e2c5d142c5539fb5e0166837013" }
-warp-ipfs = { git = "https://github.com/Satellite-im/Warp", rev = "786328d0c7ef6e2c5d142c5539fb5e0166837013" }
-warp-blink-wrtc = { git = "https://github.com/Satellite-im/Warp", rev = "786328d0c7ef6e2c5d142c5539fb5e0166837013" }
+warp = { git = "https://github.com/Satellite-im/Warp", rev = "ac30b999b7b1ea0e4f3ca81e16766590c722e9c2" }
+warp-ipfs = { git = "https://github.com/Satellite-im/Warp", rev = "ac30b999b7b1ea0e4f3ca81e16766590c722e9c2" }
+warp-blink-wrtc = { git = "https://github.com/Satellite-im/Warp", rev = "ac30b999b7b1ea0e4f3ca81e16766590c722e9c2" }
```
Comment (dariusc93) [inline Cargo.toml]:
```suggestion
warp = { git = "https://github.com/Satellite-im/Warp", rev = "06a689c69c12b126d4266020d97a74bd74c2fc3e" }
warp-ipfs = { git = "https://github.com/Satellite-im/Warp", rev = "06a689c69c12b126d4266020d97a74bd74c2fc3e" }
warp-blink-wrtc = { git = "https://github.com/Satellite-im/Warp", rev = "06a689c69c12b126d4266020d97a74bd74c2fc3e" }
```

File: ui/src/layouts/log_in/enter_seed_words.rs [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                    }
+                };
+
+                match rsp {
+                    Ok(ident) => {
+                        page.set(AuthPages::Success(ident));
+                    }
+                    Err(e) => {
+                        loading.set(false);
+                        log::error!("warp runner cmd cancelled: {e}");
```
[the remark this answers is not in the document]
Comment (sdwoodbury) [reply ui/src/layouts/log_in/enter_seed_words.rs:74]: stop saying the British spelling is wrong. go read a book or two.

**Comment by InfamousVague** [comment]
image
Added styling

**Comment by InfamousVague** [comment]
https://github.com/Satellite-im/Uplink/assets/1770198/449bec69-43a1-417a-a5cd-494dedf2274a

Finished UI Tweaks.

**Pull request merged**